<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemSettingDialog</class>
 <widget class="QDialog" name="SystemSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>423</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微軟正黑體</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>System Setting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*

QWidget
{
	background:  rgba(32, 80, 96, 100);
	color: white;
}

QLabel, QCheckBox
{ 
	color: white; 
	border-radius: 3px;
}


QPushButton, QComboBox
{
	color: white;
	background-color: #27a9e3;
	border-width: 0px;
	border-radius: 3px;
}

QPushButton:hover, QComboBox:hover, QPushButton:checked{ background-color: #66c011; }

QTableView
{
	background:  rgba(32, 80, 96, 100);
	color: white;
}

QHeaderView::section { color:black }

*/

QDialog, QRadioButton , QLabel, QCheckBox
{ 
	background:  rgba(32, 80, 96, 100);
	color:white;
}

QPushButton
{
	color: white;
	background-color: #27a9e3;
	border-width: 0px;
	border-radius: 3px;
}

QPushButton:hover, QComboBox:hover{ background-color: #66c011; }
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
   <item row="4" column="0">
    <widget class="QLabel" name="databaseLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID Database</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="cardDbBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Card ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tableDbBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Table ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changePwdBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Change PWD</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="languageLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="langEnBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">English</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="langZhBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">繁體中文</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="resetStatusBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Reset Status</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rebootBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>System Reboot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shutdownBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>System Shutdown</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="labelPerformanceMode">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Performance Mode</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="volumeLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0">
     <item>
      <widget class="QLabel" name="lowLabel">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">0 %</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QScrollBar" name="volumeScrollBar">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="highLabel">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">100 %</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testSoundBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Img/volume.ico</normaloff>Img/volume.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="speechRecogCheckBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Speech Recognition
(when Robot is Idle)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="performanceModeBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Performance Mode</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Ultrasonic</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QGridLayout" name="lightGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Front Left</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Front Right</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rear</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Front</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Left Arm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Right Arm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_8">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Origin Setting</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="originHorizontalLayout" stretch="0,0,0,0,0,0,0">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Position X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="originDsb_x">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <double>-50.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Position Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="originDsb_y">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <double>-50.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Angle</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="originDsb_angle">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="originConfirmBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
